Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -665,34 +665,31 @@
     version: "5.2.1.1329"
     engine: WebKit
     engine_version: "532.4"
--
+- 
   user_agent: HotJava/1.1.2 FCS
   client:
     type: browser
     name: HotJava
     short_name: HJ
     version: "1.1.2"
-    engine:
+    engine: ""
     engine_version: ""
--
+- 
   user_agent: Mozilla/5.0 (X11; Linux x86_64; rv:10.0.12) Gecko/20130823 Firefox/10.0.11esrpre Iceape/2.7.12
   client:
-    type: browser
-    name: Iceape
-    short_name: IA
+    type: pim
+    name: SeaMonkey
     version: "2.7.12"
-    engine: Gecko
-    engine_version: ""
```

Comment:
The seamonkey test cases needs to be moved to the pim fixtures

Reply:
moved.